k8s/prod/common-overlay/sscs/kustomization.yaml

```diff
@@ -3,6 +3,8 @@ kind: Kustomization
 namespace: sscs
 bases:
 - ../../../namespaces/sscs
+- ../../../namespaces/sscs/sscs-cor-frontend/sscs-cor-frontend.yaml
```

Comment:
If we're doing the prod deployment should we move this into the main base?

Reply:
Files moved into main base.